omero/developers/index.rst

```diff
@@ -30,6 +30,7 @@ from |Glencoe| OMERO is released under the `GNU General Public License (GPL)`_
 with commercial licenses and customization available from |Glencoe|. You can
 read about how OMERO has developed since the project started in the
 :doc:`/users/history`.
+We ask contributors to sign a `Contributor License Agreement (CLA) <https://drive.google.com/file/d/1ld6s678JWRF15qUT4_3LyTE4IGYh-wRZ/view?usp=sharing >`_. The agreement only needs to be sign once.
```

Comment:
Maybe add an email address where to send the complete CLA too. In our docs contributors@openmicroscopy.org is mentioned for that, but I don't know where this goes, I don't get emails sent to this address.